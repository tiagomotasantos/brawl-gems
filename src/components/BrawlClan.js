import React from 'react';

const BrawlClan = ({ icon, value }) => (
    <div className="brawl-clan">
        <div className="brawl-clan__values-container">
            <span>{value}</span>
            <img alt="icon" src={icon} className="brawl-clan__icon" />
        </div>
    </div>
);

export default BrawlClan;