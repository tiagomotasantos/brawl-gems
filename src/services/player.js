
import axios from './axios';

export const getPlayer = (tag) => {
    return axios.get(`player?tag=${tag}`);
}