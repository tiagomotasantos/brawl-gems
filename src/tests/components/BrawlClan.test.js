import React from 'react';
import { shallow } from 'enzyme';
import BrawlClan from '../../components/BrawlClan';
import player from '../fixtures/player';

test('should render BrawlClan', () => {
    const wrapper = shallow(<BrawlClan 
        value={player.club.name} 
        icon={player.club.badgeUrl} 
    />);
    expect(wrapper).toMatchSnapshot();
});