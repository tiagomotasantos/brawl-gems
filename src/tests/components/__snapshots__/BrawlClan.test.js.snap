// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BrawlClan 1`] = `
<div
  className="brawl-clan"
>
  <div
    className="brawl-clan__values-container"
  >
    <span>
      LEGI√ÉO PT
    </span>
    <img
      alt="icon"
      className="brawl-clan__icon"
      src="https://www.starlist.pro/img/club/0.png"
    />
  </div>
</div>
`;
